src/test/components-test.js

```diff
@@ -9,52 +9,54 @@ import dataCanvas from 'data-canvas';
 import pileup from '../main/pileup';
 import ContigInterval from '../main/ContigInterval';
 import {waitFor} from './async';
-    
+
 describe('pileup', function() {
-  var tracks = [
```

Comment:
Does this file need to be modified? If not, can you revert it?

Reply:
This is not a pure variable - it triggers downloading files and creating complex data structure that are used only in one unit test. I transformed it to function, but I would move it completely to unit test using it.
When I was developing/testing this initializer was poluting tests that should stay unaffected.